apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -60,6 +63,8 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
 
         StringBuilder dbStatement = new StringBuilder();
         String operationName = "Redisson/";
+        String methodName = "";
+        Object[] arguments = new Object[0];
 
         if (allArguments[0] instanceof CommandsData) {
             operationName = operationName + "BATCH_EXECUTE";
```

Comment:
How about we set the command(methodName) to `BATCH_EXECUTE ` as a special command here, it will be consistent with operationName

Reply:
I agree with you.